### DISCUSSION TITLE
Prevent Opening of PowerShell Window When Customer Clicks On Toast

### DISCUSSION THREAD

**Discussion by isuwill** [Windos/BurntToast#184] [Q&A]
I'm really liking this module!  There's one behavior that's good for most of the toast notifications that a customer would see, but that's not so good when I'm using PowerShell to send them toast notifications.  I was hoping someone has already addressed this.  

When I click on the toasts notifications I've sent using New-BurntToastNotification, a PowerShell window opens.  Is it possible to prevent that behavior?  I wouldn't like my customers launching PowerShell Windows on the toasts I send their way.  Not sure if this behavior was that I used Get-StartApps to find PowerShell and specify it in the config file so I could get this working on Windows 11.  

I do see that New-BurntToastNotification cmdlet does have arguments named ActivatedAction and DismissedAction.  That's the route I was thinking of going, but it would be nice if there was a global setting or something that I could use instead.  Thanks.  

BurntToast version 0.8.5.

$psversiontable

Name                           Value
----                           -----
PSVersion                      7.1.5
PSEdition                      Core
GitCommitId                    7.1.5
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
